src/controllers/v1/entityTypes.js

```diff
@@ -6,36 +6,38 @@
  */
 
 // Dependencies
-const entityTypesHelper = require(MODULES_BASE_PATH + "/entityTypes/helper")
+const entityTypesHelper = require(MODULES_BASE_PATH + '/entityTypes/helper')
+const csv = require('csvtojson')
+const { bulkAdd } = require('../../module/entityTypes/helper')
```

Comment:
@MallanagoudaB remove this already all the function will be available in entityTypesHelper